source/PlayerInfo.cpp

```diff
@@ -105,6 +105,13 @@ namespace {
 
 
 
+PlayerInfo::PlayerInfo():
+	conditions(GameData::VaryingConditions())
```

Comment:
oh so thats what you did, its held by gamedata now
seems good that'd help me for my other PR too

Reply:
that change could be carried over to another PR if you want to get things moving
we have had this condition access issue for quite some time

Reply:
It has to be available when reading the static game data, so that's the only place I can put it.